sources/packages/backend/apps/db-migrations/src/sql/SFASRestrictions/Add-col-is-included.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+  sims.sfas_restrictions
+ADD
+  COLUMN is_included bool NOT NULL DEFAULT FALSE;
```

Comment:
Why do we have this column? and why I believe we follow keywords BOOLEAN than bool. I am ok with the change but asking to implement the existing standards followed.

Reply:
I have changed `bool` to `boolean` @guru-aot. I added this column to differentiate the pre-existing SFAS restrictions that have already been inserted into SIMS student restrictions table from the SFAS restrictions that are yet to be inserted.